extensions/wikia/Search/classes/Test/ResultTest.php

```diff
@@ -487,15 +476,11 @@ public function testOffsetGetTitle() {
 	/**
 	 * @group Slow
 	 * @slowExecutionTime 0.09947 ms
-	 * @covers Wikia\Search\Result::offsetGet
+	 * @covers \Wikia\Search\Result::offsetGet
 	 */
 	public function testOffsetGetText() {
 		$result = $this->getMock( 'Wikia\\Search\\Result', [ 'getText' ] );
-		$result
-			->expects( $this->any() )
-			->method ( 'getText' )
-			->will   ( $this->returnValue( 'my text' ) )
-		;
+		$result->expects( $this->any() )->method( 'getText' )->will( $this->returnValue( 'my text' ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "my text" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)